strings/names/moth_last.txt

```diff
@@ -1,13 +1,10 @@
 Accurata
-Adultera
```

Comment:
:+1:

Reply:
For this one I thought it was weird we'd have "Adulteress" in Latin as a name, if that's okay with y'all then it still follows the guidelines besides that.

Reply:
This is a certified latin moment I would say, since there's a Moth species that's exactly that, https://en.wikipedia.org/wiki/Catocala_adultera, and while yes, Adultera means literally "An adulteress", it's only contextualized to moths in this case.

Still won't likely save it i believe.